dev/nm/datarepo-api.yaml

```diff
@@ -19,6 +17,10 @@ env:
   TERRA_COMMON_TRACING_STACKDRIVER_EXPORT_ENABLED: false
   OPENCENCUS_SPRING_ENABLED: false
   GOOGLE_SECURE_FOLDER_RESOURCE_ID: "753276429356"
+  OIDC_CLIENTID: 970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercontent.com
```

Comment:
Curious, would you advise that others update their personal yamls as well?

Reply:
That's a good point.  I'll update the personal deployments too (note: those need to be manually redeployed by the namespace owners